<?xml version="1.0"?>
<launch>

  <!-- Start publishing the map->odom transform -->
  <node name="pub_map_odom_tf" pkg="robot_planning" type="pub_map_odom_tf" />
  
  <!-- Run the costmap node -->
  <node name="costmap_node" pkg="costmap_2d" type="costmap_2d_node" >
    <rosparam file="$(find robot_planning)/launch/costmap_param/global_costmap_params.yaml" command="load" />
  </node>

</launch>